src/_layouts/default.html

```diff
@@ -0,0 +1,18 @@
+---
+layout: base.html
+---
+
+  <div class="site-wrapper">
+
+    {% include site-header-nav.html  %} 
+  
+    <main>
```

Comment:
💡 A simple improvement you could make to your markup would be to add a dynamic class to your `main` element here, which would allow you to avoid using extra `div`s in your markup to identify which page you're on. So, for example, if you did something like this:

```liquid
<main class="{{ pageTitle | slug }}-page">
```

The [`slug` filter](https://www.11ty.dev/docs/filters/slug/) is provided by Eleventy and will take your `pageTitle`, make it lowercase and replace all spaces with hyphens.

So, now each page has a unique class for you to use when styling it, e.g.:
- `contact-page`
- `routes-page`
- `service-page`

This would replace the need for extra markup and classes like `site-grid-contact`.

If using `pageTitle` doesn't work for you, you could instead add a different name to each page's front matter and use that.

In your HTML:
```yaml
---
pageTitle: Page Not Found
pageClass: error
---
```

And then in this layout file:
```liquid
<main class="{{ pageClass | slug }}-page">
```

would result in:

```html
<main class="error-page">
```